### DISCUSSION TITLE
AOD always on 120hz on A53

### DISCUSSION THREAD

**Discussion by Xisrr1** [tribalfs/GalaxyMaxHzPub#170] [Q&A] [answered]
Is it a bug?

**Comment by tribalfs** [comment]
What version do you have? Did you check it using the refresh rate monitor from the developer options?

**Comment by Xisrr1** [comment]
The latest version, 8.1, and I checked with the refresh rate monitor.

**Comment by tribalfs** [comment]
It should be [v8.1.30](https://github.com/tribalfs/GalaxyMaxHzPub/releases). Use the refresh rate monitor from developer option to check.

**Comment by Xisrr1** [comment]
Yes, I'm using this version. AOD is on 120hz. I checked several times.

**Comment by tribalfs** [comment]
What device is this?
Did you check using the "show refresh rate" option in the Developer options?

**Comment by Xisrr1** [comment]
Galaxy A53. Yes, I've checked using the refresh rate monitor. Only if I lock the screen when it's on 60hz it will stay on 60hz on AOD.

**Comment by tribalfs** [comment]
From developer options?

**Comment by Xisrr1** [comment]
Yes

**Comment by Xisrr1** [comment]
Yes

**Comment by Xisrr1** [comment]
I think GMH failes to switch the refresh rate when the screen is off.

**Comment by tribalfs** [comment]
It does actually changed but seems the refresh rate monitor does not update immediately anymore on AOD, not until the AOD shifts position (try to wait a  bit for it).

You can also check it using ADB command while on AOD/screen off:
`adb shell settings get system peak_refresh_rate`

**Reply by Xisrr1** [reply]
When the screen is off adb says "null".

**Reply by tribalfs** [reply]
Ensure you did not enable "ignore while charging"

**Reply by Xisrr1** [reply]
Where is that setting?

**Reply by tribalfs** [reply]
Inside screen-off/AOD refresh rate

**Reply by Xisrr1** [reply]
I do not use psm when screen off ![Screenshot_20240411_161630_Galaxy MaxHz.png](https://github.com/tribalfs/GalaxyMaxHzPub/assets/133952156/61ca28f0-c40f-4c8b-abdb-6c943055f4f3)

**Reply by tribalfs** [reply]
No, not that setting. I mentioned it's under "screen-off/AOD refresh rate" but available only for premium. By default, screen/aod off settings only applies when not plugged/charging.

**Reply by Xisrr1** [reply]
![Screenshot_20240411_162256_Galaxy MaxHz.png](https://github.com/tribalfs/GalaxyMaxHzPub/assets/133952156/ff093c0a-9cf2-44a2-b549-9562c607c0c0)

So this?

**Reply by tribalfs** [reply]
No, that's for auto PSM on screen off.

**Reply by Xisrr1** [reply]
I'm confused.

**Reply by tribalfs** [reply]
![Screenshot_20240411_212518_Galaxy MaxHz](https://github.com/tribalfs/GalaxyMaxHzPub/assets/65062033/d8f3463b-55ac-4465-891b-edb91b0ad21d)

**Reply by Xisrr1** [reply]
![Screenshot_20240411_161630_Galaxy MaxHz.png](https://github.com/tribalfs/GalaxyMaxHzPub/assets/133952156/74bfdcb6-cc7e-4501-8d94-bbdac259ef4e)

It is not customizable for me?

**Reply by tribalfs** [reply]
It's a bug the option not showing on your device. 
Please update to [v8.1.50](https://github.com/tribalfs/GalaxyMaxHzPub/releases/tag/v8.1#:~:text=Latest%20Build%3A%20v8.1.50)
Thank you.

**Reply by Xisrr1** [reply]
Oh a new update! Fixed

**Reply by Xisrr1** [reply]
Did you release this update just because of me? I thought it would have OneUI 6 switches.

**Reply by Xisrr1** [reply]
Hmm still 120hz on AOD.

**Reply by tribalfs** [reply]
It will come very soon. Just testing new features

https://github.com/tribalfs/GalaxyMaxHzPub/assets/65062033/64449847-0a28-4774-b611-058a7b737079

**Reply by Xisrr1** [reply]
MMMMM... Looks good

Will it support predictive back animations?

**Comment by tribalfs** [comment]
> Hmm still 120hz on AOD.


Did you use ADB to check?

**Comment by tribalfs** [comment]
> What about the 60hz on AOD still doesn't work?

I can't reproduce it on my devices. Can you send a recording that I can look into?  Pls also show your relevant settings.

**Reply by tribalfs** [reply]
Please do check in ADB too. As I mentioned previously, the refresh rate monitor may  stop reflecting the actual refresh rate (probably to save power) while on AOD and only updates when AOD position shifts

**Reply by Xisrr1** [reply]
I don't have access to a pc rn. But the refresh rate monitor from developer options is always on. And it's stays on 120hz while on AOD.

**Reply by Xisrr1** [reply]
> > What about the 60hz on AOD still doesn't work?
> 
> I can't reproduce it on my devices. Can you send a recording that I can look into?  Pls also show your relevant settings.

![Screenshot_20240414_004236_Galaxy MaxHz.png](https://github.com/tribalfs/GalaxyMaxHzPub/assets/133952156/72fb1a74-5552-43b5-b8b3-a7ad0a5f19b7)

![Screenshot_20240414_004254_Galaxy MaxHz.png](https://github.com/tribalfs/GalaxyMaxHzPub/assets/133952156/3331c339-c229-469d-bd70-74773f9f3b4a)

![Screenshot_20240414_003922_Galaxy MaxHz.jpg](https://github.com/tribalfs/GalaxyMaxHzPub/assets/133952156/e3c0627c-a8c6-4be8-80e4-d43015c97884)

**Reply by Xisrr1** [reply]
When using the refresh artwork monitor from the dev option, it always shows and is on 120hz. I can't access a PC now.

**Reply by tribalfs** [reply]
The refresh rate monitor may stop reflecting the actual refresh rate  when screen is off.

**Reply by Xisrr1** [reply]
It's not. It shows on the always-on display and doesn't disappear, so I think it is correct.

I won't die using 120hz on AOD, but it would be nice to be fixed.

**Reply by tribalfs** [reply, marked as answer]
What I meant is it still showing but "STOPS UPDATING" real time when screen is off or AOD.
You have to double check it using ADB.

**Reply by Xisrr1** [reply]
I'll check and update today.

**Comment by Xisrr1** [comment]
I shared screen shots

**Comment by tribalfs** [comment]
https://github.com/tribalfs/GalaxyMaxHzPub/discussions/170#discussioncomment-9106223
